lib/american_date_parsing.rb

```diff
@@ -31,9 +31,9 @@ def parse_as_americanized_date(*attributes)
 
       define_method "#{attribute}=" do |date_string|
         parsed = if date_string.respond_to?(:strftime)
```

Comment:
Anyone know why this checked if it responds to `strftime` instead of just `to_date` (or now `to_datetime`)?